storage-proofs-porep/src/stacked/vanilla/proof.rs

```diff
@@ -440,15 +451,21 @@ impl<'a, Tree: 'static + MerkleTreeTrait, G: 'static + Hasher> StackedDrg<'a, Tr
     // Even if the column builder is enabled, the GPU column builder
     // only supports Poseidon hashes.
     pub fn use_gpu_column_builder() -> bool {
+        // TODO (halo): change `PoseidonHasher<Fr>` to
```

Comment:
Note: this needs to be updated once GPU Poseidon is supported for the Pasta fields.